Assets/Scripts/Bee/beeTutorial.cs

```diff
@@ -5,76 +5,178 @@
 
 public class beeTutorial : MonoBehaviour
```

Comment:
Are we still using this? There seems to be individual scripts for each panel now right?